src/components/Activity/DefaultTabBar.js

```diff
@@ -47,29 +55,28 @@ class DefaultTabBar extends Component {
       height: 0.5,
       borderColor: '#f0f0f2',
       borderBottomWidth: 1,
-      bottom: 0,
+      bottom: 0.5,
       zIndex: -1,
     };
 
     const left = this.props.scrollValue.interpolate({
-      inputRange: [0, 1, ],
-      outputRange: [this.props.leftOffset,  (containerWidth / 2) - this.props.rightOffset, ],
+      inputRange: [0, 1],
+      outputRange: [this.props.leftOffset,  (containerWidth / 2) - this.props.rightOffset],
     });
 
     return (
       <View>
-        <View
-          style={[
-            styles.tabs,
-            { backgroundColor: this.props.backgroundColor, },
-            this.props.style,
-            ]}>
+        <View style={[
+          styles.tabs,
+          { backgroundColor: this.props.backgroundColor, },
+          this.props.style,
+          ]}>
```

Comment:
73:11 Expected indentation of 8 spaces but found 10
73:13 The closing bracket must be aligned with the line containing the opening tag (expected column 9 on the next line)